packages/web-components/src/components/video-player/video-player-composite.ts

```diff
@@ -42,7 +42,19 @@ class DDSVideoPlayerComposite extends HybridRenderMixin(HostListenerMixin(LitEle
    *
    * @internal
    */
-  _embedMedia?: (videoId: string) => Promise<any>;
+  _embedMedia?: (videoId: string, backgroundMode: boolean) => Promise<any>;
```

Comment:
@andy-blum looks like this is a new boolean added which is causing errors in the downstream app.

@annawen1 @andysherman2121 in these cases would there be a default value set if it's not defined?

Reply:
@jeffchew there's a default value on the `backgroundMode` parameter, but it's defined in video-player-container. I've updated the typescript for this file to correctly mark `backgroundMode` as optional.